packages/terra-open-source-scripts/src/terra-cli/express-server/ExpressServer.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs-extra');
```

Comment:
Is this essentially a copy paste of the terra-functiona-testing express server?

Reply:
Yes. @mjhenkes and I discussed this briefly and decided it might be best to do that until we land on a long term location for that server. It felt weird to have it exposed from functional tests or to have a dependency of this script on functional tests. This is a temporary solution